src/test/java/teammates/test/cases/automated/SubmissionsAdjustmentTest.java

```diff
@@ -225,8 +225,7 @@ public void testEnrollStudentsWithScheduledSubmissionAdjustment() throws Excepti
         } catch (EnrollException e) {
             String actualErrorMessage = e.getLocalizedMessage();
 
-            String errorReason = String.format(FieldValidator.EMAIL_ERROR_MESSAGE, "e6@g@",
-                    FieldValidator.REASON_INCORRECT_FORMAT);
```

Comment:
The invalid input `e6@g@` is not displayed in the simplified error message (only the field name).
`REASON_INCORRECT_FORMAT` is also removed because it is sort of redundant: it is defined as `is not in the correct format`